src/java.base/unix/native/libjli/java_md_common.c

```diff
@@ -108,6 +110,47 @@ GetApplicationHomeFromDll(char *buf, jint bufsize)
     return JNI_FALSE;
 }
 
+#if defined(AIX)
+static jboolean
+LibjavaExists(const char *path)
+{
+    char tmp[PATH_MAX + 1];
+    struct stat statbuf;
+    JLI_Snprintf(tmp, PATH_MAX, "%s/%s", path, JAVA_DLL);
+    if (stat(tmp, &statbuf) == 0) {
+        return JNI_TRUE;
+    }
+    return JNI_FALSE;
+}
+
+/*
+ * Retrieves the path to the JRE home by locating libjava.so in
+ * one of the LIBPATH and then truncating the path to it.
```

Comment:
```suggestion
 * LIBPATH and then truncating the path to it.
```

Small comment suggestion